### PR TITLE
waithelpers AccountAddOnsCalendarDashboardFilesHelpdeskPages

### PR DESCRIPTION
**Pull request by alstanki**
# Description
switch to use waitHelpers: Account, AddOns, Calendar, Dashboard, Files, Helpdesk Pages

## Links to Tickets or other pull requests
https://ticketsystem.hpi-schul-cloud.org/browse/SC-6939

## Changes
Methods from waitHelpers library to Account, AddOns, Calendar, Dashboard, Files, Helpdesk Pages were applied
Removed outdated and incorrectly written/duplicated - methods/tests/steps

## Approval for review
- [ ] All points were discussed with the ticket creator, support-team or product owner. The code upholds all quality guidelines from the PR-template.

> Notice: Please remove the WIP label if the PR is ready to review, otherwise nobody will review it.

### Link to Definiton of Done
More and detailed information on the *definition of done* can be found [on Confluence](https://docs.schul-cloud.org/pages/viewpage.action?pageId=92831762)

### CODE REVIEW

File: features/visibilityOfMenuItems.feature [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+        Given <userRole> arrives on the Schul-Cloud homepage
+        And <userRole> logs in
+        And <userRole> accepts data protection
+        When click left navigation item "administration"
+        Then <userRole> should see that all sub menu items are visible: <tabsList> in administration area
+
+        Examples:
+            | userRole | tabsList                                       |
+            | admin    | SCHÜLER, LEHRER, KURSE, KLASSEN, TEAMS, SCHULE |
+            | teacher  | SCHÜLER:INNEN, LEHRER:INNEN, KLASSEN           |
```
Comment (d3n153) [inline features/visibilityOfMenuItems.feature:19]: Would be nice, if you could add a new line.

File: page-objects/pages/HMWRKHomeworkListPage.js [hunk 2]
```diff
@@ -85,24 +84,26 @@ module.exports = {
...
-        return isTaskOnList;
+        const msg = 'Task with name is not visible on the list: \n';
+        const resultMsg = 'Expected: ' + taskname + ', Actual: ' + allTasks;
+        await expect(isTaskOnList, msg + resultMsg).to.equal(true);
+    },
+
+    isTaskNotVisible: async function (taskname) {
+        const allTasks = await this.getListOfTaskTitles();
+        const isTaskOnList = allTasks.some((element) => element.includes(taskname));
+        const msg = 'Task with name is not visible on the list: \n';
```
Comment (d3n153) [inline page-objects/pages/HMWRKHomeworkListPage.js:104]: Message should be different in this case, or?

**Review by d3n153 — APPROVED** [review]
Since Alicja will fixed the found improvements within her next PR, I'll approve this one.

**Review by maugosiam — APPROVED** [review]

File: page-objects/pages/HMWRKHomeworkListPage.js (hunk 2 above)
Comment (alstanki) [reply page-objects/pages/HMWRKHomeworkListPage.js:104]: Yes I will fix it today

**Pull request merged**